src/python/supply/supply.go

```diff
@@ -591,6 +591,8 @@ func (s *Supplier) UninstallUnusedDependencies() error {
 }
 
 func (s *Supplier) RunPipUnvendored() error {
+	s.Log.BeginStep("Running Pip Install (Vendored)")
```

Comment:
I think you got the 2 logging steps mixed up

Reply:
indeed, sloppy; fixed now.